mathics/builtin/numbers/algebra.py

```diff
@@ -405,44 +403,59 @@ class Simplify(Builtin):
 
     Simplify over conditional expressions uses $Assumptions, or $assump$
     to evaluate the condition:
-    # TODO: enable this once the logic for conditional expression
-    # be restaured.
-    # >> $Assumptions={a <= 0};
-    # >> Simplify[ConditionalExpression[1, a > 0]]
-    # = Undefined
-    # >> Simplify[ConditionalExpression[1, a > 0], { a > 0 }]
-    # = 1
+    >> $Assumptions={a <= 0};
+    >> Simplify[ConditionalExpression[1, a > 0]]
+     = Undefined
+    >> Simplify[ConditionalExpression[1, a > 0], Assumptions -> { a > 0 }]
```

Comment:
In Wolfram Language this is `SimplifyConditionalExpression[1, a > 0], { a > 0 }]`. But for me, having an option instead of a parameter is better.

Reply:
This is the documentation of `Simplify` in WMA 12:

```
Simplify[expr] performs a sequence of algebraic and other\
         
        >     transformations on expr
         
        >     and returns the simplest form it finds. Simplify[expr, assum]

         
        >      does simplification using assumptions. 


        Attributes[Simplify]={Protected}


        Options[Simplify]={Assumptions :> $Assumptions, 
         
        >    ComplexityFunction -> Automatic, ExcludedForms -> {}, 
         
        >    TimeConstraint -> 300, TransformationFunctions -> Automatic, 
         
        >    Trig -> True}
```

Reply:
However, it is true: ``Simplify[expr, assump]`` is equivalent to ``Simplify[expr, Assumptions->assump]``

Reply:
I'd be grateful if you'd go over our Simplify doc that we have to expand the explanation for the expanded behavior. 

(That reminds me that I have been meaning to revise our doc for Expression to make it clear that in the user docs we are talking about the symbol rather than the data structure. I'll be doing that soon.)